test/runtests.jl

```diff
@@ -335,7 +335,7 @@ OBJACOORD
 
 FCOORD
 1
-0 0 0 1 -0.9999999999999998
+0 0 0 1 -0.9999999999999999
```

Comment:
No, you might loose precision. It only works for special cases like with 1.0.
For instance,
```julia
(pi * sqrt(2)) / sqrt(2) == pi # -> false
```
But here it seems you have -1.0 so I would have expected it to work